data/tmpl/values/content3.html

```diff
@@ -32,7 +32,11 @@
             </label>
           </li>
         </ul>
-        <button class="button">Tell Me</button>
+        <button id="tell_me" class="button">Tell Me</button>
+        <section id="private_browsing" class="hidden">
```

Comment:
It is kinda tricky, as clicking the first button needs to show the answer but, also show the additional content. This is what this does currently. One thing we can do, is to have the content kinda slide down or fade in, instead of the current jerky interaction.

Definitely something we should talk to Matt about but, fine tuning we can easily do once all the sidebars and their base functionality has landed.